build.savant

```diff
@@ -170,6 +170,9 @@ target(name: "build-java", description: "Build the Java Client Library") {
     filter(token: ".*@MaskMapValue\\(.*\n", value: "")
     filter(token: ".*@MaskString.*\n", value: "")
     filter(token: ".*@InternalUse.*\n ", value: "")
+    // don't want internal tickets and such in here
```

Comment:
Seemed like we may not want this in the client code. not a huge deal but not  perhaps not ideal.